packages/block-library/src/heading/style.scss

```diff
@@ -11,3 +11,14 @@ h6 {
 		padding: $block-bg-padding--v $block-bg-padding--h;
 	}
 }
+
+:root {
+	h1,
+	h2,
+	h3,
+	h4,
+	h5,
+	h6 {
+		line-height: var(--wp--typography--line-height);
```

Comment:
Should we move this here instead https://github.com/WordPress/gutenberg/pull/21215/files#diff-f2f5efb13e0e16e61312aa9b0c028ae1R8

Reply:
@youknowriad The link doesn't appear to be working for me 😊

Reply:
a few lines up in the same file (with the other variables) :)

Reply:
Ah. Something like this?

```
:root {
	h1,
	h2,
	h3,
	h4,
	h5,
	h6 {
		line-height: var(--wp--typography--line-height);
	}
}

h1,
h2,
h3,
h4,
h5,
h6 {
	background-color: var(--wp--color--background);
	color: var(--wp--color--text);

	&.has-background {
		padding: $block-bg-padding--v $block-bg-padding--h;
	}
}
```

Reply:
No, I think I missed the `:root` thing when I merged the heading PR. I was thing more like this:

```
:root {
	h1,
	h2,
	h3,
	h4,
	h5,
	h6 {
		background-color: var(--wp--color--background);
		color: var(--wp--color--text);
		line-height: var(--wp--typography--line-height);
	}
}

h1,
h2,
h3,
h4,
h5,
h6 {
	&.has-background {
		padding: $block-bg-padding--v $block-bg-padding--h;
	}
}
```

Reply:
Ah gotcha! Thank you for clarifying :). I'll make those updates